### PR TITLE
ESRI Arcgis Feature Server connector

### PR DESCRIPTION
**Pull request by fvankrieken**
Add a new connector for [ESRI Arcgis Feature Service](https://developers.arcgis.com/rest/services-reference/enterprise/feature-service.htm) and use it in library. Keeping this as only part of #672 so that others can use this connector

[Working action](https://github.com/NYCPlanning/data-engineering/actions/runs/8266756531)

### CODE REVIEW

File: dcpy/connectors/esri/arcgis_feature_server.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+from pydantic import BaseModel
+from typing import cast
+
+
+class Server(StrEnum):
+    nys_clearinghouse = "nys_clearinghouse"
+
+    @property
+    def id(self):
+        match self:
```
Comment (fvankrieken) [inline dcpy/connectors/esri/arcgis_feature_server.py:12]: This is silly at the moment, I'll just make a dict and non-class function. Will probably play nicer with pydantic anyways

File: docker/config.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # Logic in this file is used within builds of docker containers
 
-COMMON_APT_PACKAGES="curl zip unzip git wget ca-certificates lsb-release build-essential sudo postgresql-client-15 libpq-dev jq locales"
+COMMON_APT_PACKAGES="curl zip unzip git wget ca-certificates lsb-release build-essential sudo postgresql-client-15 libpq-dev jq locales libkrb5-dev"
```
Comment (fvankrieken) [inline docker/config.sh:5]: Needed to install arcgis python package. Even if we don't go that route, might leave, it's a pretty insignificant package to include

**Comment by fvankrieken** [comment]
Going to incorporate this in extract in my extract PR follow-up to #654

**Review by alexrichey — APPROVED** [review]
Cool - looking forward to giving it a spin

**Comment by damonmcc** [comment]
nice! and looks like no need for an esri python package?

**Comment by fvankrieken** [comment]
Not for now at least. `arcgis` is actually pretty nice to use, but this was easy enough to implement (thanks for some points in the right direction @jackrosacker), and `arcgis` had a ton of dependencies, so seemed tidy just to keep things lighter-weight for the time being

**Pull request merged**